hyperspy/tests/signal/test_apodiztion.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2007-2016 The Hyperspy developers
+#
+# This file is part of  Hyperspy.
+#
+#  Hyperspy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+#  Hyperspy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with  Hyperspy.  If not, see <http://www.gnu.org/licenses/>.
+
+import pytest
+import numpy as np
+from numpy.testing import assert_allclose
+
+from hyperspy.signals import Signal1D, Signal2D, ComplexSignal1D, ComplexSignal2D, BaseSignal
+
+
+def _generate_parameters():
+    parameters = []
+    for lazy in [False, True]:
+        for window_type in ['hann', 'hamming']:
+            parameters.append([lazy,
+                               window_type])
+    return parameters
+
+
+@pytest.mark.parametrize('lazy, window_type', _generate_parameters())
+def test_apodization(lazy, window_type):
```

Comment:
test looks good. Can we add something similar for the fft case? Using the holo dataset